src/components/button-favourite/button-favourite.test.ts

```diff
@@ -0,0 +1,46 @@
+describe("Favorite button", () => {
+  it("Material id is not on user's favorites list", () => {
```

Comment:
Consider following the textual convention á la:

```suggestion
  it("Adds a new material to the favorites list when clicked", () => {
```